### ISSUE TITLE
Task 'wrapper' not found in project ':app'.

### ISSUE THREAD

**Issue by phillip9077** [googlesamples/mlkit#292]
Running the Android version of this app and gradle will not build. It throws the error message as mentioned in the title. Tried adjusting the targetSdkVersion and adding

`task wrapper(type: Wrapper){
   gradleVersion = '2.14.1'
}`

to the gradle file to no avail. Any help would be appreciated!

Edit:
I'm trying to open the source code for the ML Kit demos in Android Studio. I have also tried opening the source code from the vision-quickstart folder rather than the app folder. The previous error disappeared, but a new one appeared:

"Could not resolve all dependencies for configuration ':app:proguardRuntimeClasspath'.
Could not create task ':app:minifyProguardWithR8'.
Cannot query the value of this provider because it has no value available."

**Comment by phillip9077** [comment]
I managed to fix the issue as it seems like the Android app requires a specific version of the Android SDK Build-tools. I installed a fresh copy of Android Studio so it automatically installed version 30.03 Android SDK Build-tools. However, the app requires version 29.0.2. 

I think an update to address this would be warranted in case a person new to Android app development is trying to run this program. It will help streamline the experience :)
